tests/RouteTest.php

```diff
@@ -216,6 +219,96 @@ public function testMiddlewareStackInterrupted(): void
         $this->assertSame(403, $response->getStatusCode());
     }
 
+    public function testInvalidMiddlewareAddWrongStringLL()
```

Comment:
What is the `LL` on the ending of method names?

Reply:
I think lazy loading